webpage/src/.vuepress/config.js

```diff
@@ -469,65 +468,67 @@ module.exports = {
           }
         ]
       },
-      '/fa/': {
-        selectText: 'زبان ها',
-        label: 'فارسی',
-        editLinkText: 'برای بهبود این سایت به ما کمک کنید!',
-        algolia: {
-          apiKey: '35f878f4b712d5ab6a659eb0a6c71576',
-          indexName: 'qownnotes'
-        },
-        nav: [
-          {
-            text: 'شروع شدن',
-            link: utils.getNavItemForLanguage(gettingStartedNavItem, 'fa'),
-          },
-          {
-            text: 'نصب و راه اندازی',
-            link: utils.getNavItemForLanguage(installationNavItem, 'fa'),
-          },
-          {
-            text: 'تغییرات',
-            link: changelogNavItem,
-            collapsable: true,
-          },
-          {
-            text: 'مخاطب',
-            ariaLabel: 'Contact Menu',
-            items: contactNavItems
-          },
-          {
-            text: 'اهدا کردن',
-            link: utils.getNavItemForLanguage(donateNavItem, 'fa'),
-          }
-        ],
-        sidebar: [
-          {
-            title: 'شروع شدن',
-            children: utils.getNavItemsForLanguage(gettingStartedNavItems, 'fa')
-          },
-          {
-            title: 'نصب و راه اندازی',
-            children: utils.getNavItemsForLanguage(installationNavItems, 'fa')
-          },
-          {
-            title: 'ویرایشگر',
-            children: utils.getNavItemsForLanguage(editorNavItems, 'fa')
-          },
-          {
-            title: 'اسکریپت ها',
-            children: utils.getNavItemsForLanguage(scriptingNavItems, 'fa')
-          },
-          {
-            title: 'مشارکت',
-            children: utils.getNavItemsForLanguage(contributingNavItems, 'fa')
-          },
-          {
-            title: 'سالات متداول',
-            children: utils.getNavItemsForLanguage(faqNavItems, 'fa')
-          }
-        ]
-      },
+‚/fa/': {
```

Comment:
All `'` characters seem to be broken and the indent is off too.

Reply:
Ok

Reply:
Can you please provide an actual **config.js** before I started working on the persian block, please?

Reply:
Because the link that you'd sent in #2045
https://github.com/pbek/QOwnNotes/blob/666eeb204c9f4ec7996521e532982b55ec4e492f/webpage/src/.vuepress/config.js#L236-L267
has been already changed, even I don't know how.

Reply:
The actual block you see on the left side of https://github.com/pbek/QOwnNotes/pull/2051/files. You already edited the correct block, but it seems you copied the text out of the text file and edited it in MS Word or something, which destroyed at lot of the original text. 😅

Reply:
Ok. I did that. :octocat: 
Will start again.

Reply:
I pasted the farsi block now below because I had the problem that I didn't manage to fork the original config.js without the changes that I've already made on the file, maybe because my earlier attempts overwrote the new ones.

Reply:
And the indent.. after editing the block in QON :/

Reply:
Confusing atm but I'll try on.